Tests/XCUITests/ActivityStreamTest.swift

```diff
@@ -124,8 +124,8 @@ class ActivityStreamTest: BaseTestCase {
         navigator.performAction(Action.CloseURLBarOpen)
         navigator.performAction(Action.OpenNewTabFromTabTray)
 
-        waitForExistence(app.collectionViews.cells.staticTexts[newTopSite["bookmarkLabel"]!])
-        XCTAssertTrue(app.collectionViews.cells.staticTexts[newTopSite["bookmarkLabel"]!].exists)
+        waitForExistence(app.collectionViews.cells.staticTexts[newTopSite["topSiteLabel"]!], timeout: TIMEOUT)
```

Comment:
Chagining `bookmarkLabel` with topSiteLabel` make the test fails here

Reply:
🤔 I would not commit anything that does not pass on my end. I found that `topSiteLabel` instead of `bookmarkLabel` has been displayed on my end. Displaying `bookmarkLabel` at this point makes more sense.

Reply:
When I run the test locally, sometimes the text is "Mozilla" but "Internet for people, not profit — Mozilla" other times. 😕 It's not an issue on M1. 🤔 💭 Let me think what I can do.

Reply:
Does the test fail consistently because of this line on your end? This test fails over 95% of the time without this change.

Reply:
This is what I see on my end. 📺 😕 
https://user-images.githubusercontent.com/1740517/202366876-f3456607-f0ac-4390-b114-1f6f9feb0993.mov